docker/docker-compose.yml

```diff
@@ -10,7 +10,8 @@ services:
     env_file:
       - ${PWD}/${ENVIROMENT}/api/.env_validator
     ports:
-      - "127.0.0.1:8080:8080" 
+      - "0.0.0.0:8080:8080" 
+      - "0.0.0.0:443:443" 
```

Comment:
We open both 8080 and 443. The validator will be serving only in one depending on its TLS cert config. But we do both bindings in the compose file so both are fine to be reached.